src/Uno.UI.Foldable/FoldableHingeAngleSensor.Android.cs

```diff
@@ -44,17 +45,24 @@ public class FoldableHingeAngleSensor : INativeHingeAngleSensor
 		}
 
 		public FoldableHingeAngleSensor(object owner)
+		{
+			_hingeSensor = GetHingeSensor();
+		}
+
+		internal static bool HasHinge => GetHingeSensor() is not null;
+
+		private static Sensor GetHingeSensor()
 		{
 			if (!(ContextHelper.Current is Activity currentActivity))
 			{
 				throw new InvalidOperationException("FoldableHingeAngleSensor must be initialized on the UI Thread");
 			}
 
-			_sensorManager = SensorManager.FromContext(currentActivity);
+			_sensorManager ??= SensorManager.FromContext(currentActivity);
 
 			var sensors = _sensorManager.GetSensorList(Android.Hardware.SensorType.All);
```

Comment:
Do we need a null check for `_sensorManager` here? 

https://docs.microsoft.com/en-us/dotnet/api/android.hardware.sensormanager.fromcontext?view=xamarin-android-sdk-12